gokrazy/go.mod

```diff
@@ -0,0 +1,25 @@
+module tailscale.com/gokrazy
```

Comment:
This is somewhat regrettably its own module, just so I can use `replace` directives at the bottom without infecting users of `tailscale.com` for e.g. `tsnet`.

Hopefully we can get all our changes upstreamed and remove this.